schema.sql

```diff
@@ -1,5 +1,8 @@
-/* Database schema to keep the structure of entire database. */
-
 CREATE TABLE animals (
-    name varchar(100)
-);
+    id INT,
```

Comment:
- [ ] [optional] You could also make your id the primary key, and make it not null and auto increment.